### PR TITLE
Create xdebug-and-phpstorm.md

### PR DESCRIPTION
**Pull request by theMikeD**

### CODE REVIEW

File: docs/en-US/references/xdebug-and-phpstorm.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+> What follows is opinionated defaults, so adjust as you deem necessary. Specifically, PHP versions you'll be supporting
+
+## Confirm phpStorm's project setup
+
+Make sure you have your project folders already set up in phpStorm.
+
+1. Click PHPStorm, Preferences, Directories
+2. Make sure your project's content root is there
+3. Click Languages & Frameworks, PHP and fill it in like this
+   - PHP language level: 7.3
```
Comment (Mte90) [inline docs/en-US/references/xdebug-and-phpstorm.md]: We need to write that is better to align this value with the one inside the `config.yml` for the specific domain. If a custom one is not defined the value is the php default version in the machine that now is php 7.2.

**Review by Mte90 — CHANGES_REQUESTED** [review]

**Comment by tomjn** [comment]
I've made some adjustments so it would work with jekyll if it was merged, and fixed some formatting stuff where block quotes were used for emphasis, etc, but I need to sit down and properly read through, there's some stylistic changes that'll be needed so it fits in with the rest of the site better.

@theMikeD thanks for getting us most of the way :) Feel free to take bits back for internal use

**Comment by theMikeD** [comment]
I figured the tone would have to change but you're more than welcome!

**Review by tomjn — APPROVED** [review]

**Comment by tomjn** [comment]
It could use some adjustment still, but it's useful enough to be on the site. I'd need a PHPStorm install to make more progress

**Pull request merged**